src/components/tooltip/Marker.js

```diff
@@ -105,6 +105,11 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
     const page = pages.find(page => page.innerWindowID === innerWindowID);
 
     if (page) {
+      // If multiple pages have the same url, show the innerWindowID to disambiguate.
+      let innerWindowIDSuffix = null;
+      if (pages.filter(p => p.url === page.url).length > 1) {
+        innerWindowIDSuffix = '(' + innerWindowID + ')';
```

Comment:
nit: I believe we need a space at the start of this string.
![image](https://user-images.githubusercontent.com/454175/114435619-a8092900-9bc4-11eb-9dbc-9631e261a3dd.png)
And then you need to remove the extra space in the error case below.